src/streams/TopicItem.js

```diff
@@ -40,8 +45,31 @@ export default function TopicItem(props: Props) {
     onPress,
   } = props;
 
+  const showActionSheetWithOptions: ShowActionSheetWithOptions = useActionSheet()
+    .showActionSheetWithOptions;
+  const _ = useContext(TranslationContext);
+  const dispatch = useDispatch();
+  const backgroundData = useSelector(state => ({
+    auth: getAuth(state),
+    mute: getMute(state),
+    subscriptions: getSubscriptions(state),
+    ownUser: getOwnUser(state),
+    flags: getFlags(state),
+  }));
+
   return (
-    <Touchable onPress={() => onPress(stream, name)} onLongPress={() => showToast(name)}>
+    <Touchable
+      onPress={() => onPress(stream, name)}
+      onLongPress={() => {
+        showHeaderActionSheet({
```

Comment:
Currently this is actually only a `TopicActionSheet`, there's no way (that I know of) to call it with just a stream and no topic.

I agree that we should rename it, but I'd prefer to punt on that until we figure out if it will in fact turn into a `StreamOrTopicActionSheet`, or if we're going to have it turn into a `TopicActionSheet`, and grow a `StreamActionSheet` as well.

I suspect it'll be the latter, but I don't think I know for sure right now, so I'd rather wait to rename until I'm more sure what this will end up looking like, to save us the effort of possibly renaming it twice.